### PR TITLE
Support ubuntu 20.04

### PR DESCRIPTION
**Pull request by Chithrak07**
Add support for ubuntu 20.04

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@
   when: ansible_distribution != 'Ubuntu'
     or (ansible_distribution_version != '14.04'
       and ansible_distribution_version != '16.04'
+      and ansible_distribution_version != '18.04'
       and ansible_distribution_version != '18.04')
```
Comment (dale-c-anderson) [inline tasks/main.yml]: 18.04 twice?

**Comment by dale-c-anderson** [comment]
@Chithrak07 does 20.04 pass molecule tests? I don't see anything changed/added there.

**Comment by Chithrak07** [comment]
@dale-c-anderson : Modified.

**Comment by dale-c-anderson** [comment]
@Chithrak07 did molecule tests against 20.04 pass?

**Comment by Chithrak07** [comment]
@dale-c-anderson Yes

 ansible-role-mariadb

**Review by dale-c-anderson — APPROVED** [review]

**Pull request merged**